GNUmakefile

```diff
@@ -46,6 +46,10 @@ acceptance: fmtcheck
 		test -n ARTIFACTORY_USERNAME && test -n ARTIFACTORY_PASSWORD && test -n ARTIFACTORY_URL \
 		&& go test -ldflags="-X '${PKG_VERSION_PATH}.Version=${NEXT_VERSION}-test'" -v -parallel 20 ./pkg/...
 
+acceptance_federated:
```

Comment:
@alexhung why do we need to run federated tests separately? If we really need this, we will need to modify CI to run them as well.

Reply:
@danielmkn Separately exactly because we need to modify Pipeline as well. This way we can separate the two activities.

Reply:
@danielmkn The env var `ARTIFACTORY_URL_2` also controls if the tests for federated repo are run or not. If the env var is set, `make acceptance` will run the tests.

This make file command is a nice-to-have to allow running those tests separately.